token-swap/program/src/curve/fees.rs

```diff
@@ -1,51 +1,77 @@
-//! Simple constant 1:1 swap curve
+//! All fee information, to be used for validation currently
```

Comment:
Yikes, I'm not sure why git thinks flat.rs -> fees.rs.  This file is new, and flat.rs became constant_price.rs